reference-architecture/gce-cli/gcloud.sh

```diff
@@ -344,6 +344,9 @@ gcloud --project "$GCLOUD_PROJECT" compute ssh "cloud-user@${OCP_PREFIX}-bastion
         --enable=\"rhel-7-server-extras-rpms\" \
         --enable=\"rhel-7-server-ose-${OCP_VERSION}-rpms\";
 
+    if ! rpm -q epel-release; then
+      yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+    fi
```

Comment:
Ansible 2.3 on bastion host is not a problem. It's problem on the host which creates the infrastructure.